dockerize-repository/docker-compose.yml

```diff
@@ -12,9 +12,7 @@ services:
             - "5432"
 
     b2share:
-        build:
-            context: ..
-            dockerfile: dockerize/Dockerfile
+        image: eudatb2share/b2share:2.0.1
```

Comment:
I am a little at loss here. If the deployment procedure is in the dockerize repository why do we need a docker-compose which also deploy production environment?

The only docker-compose which seems useful to me here is one for development, thus it should use local code. Otherwise we will be in the same situation as before the last docker scripts refactoring when we could not test our modifications before merging PRs. In fact it would be even worse here as we would have to release on docker-hub before being able to test.